src/class.cr

```diff
@@ -129,7 +129,7 @@ class Class
   # ```
   # Int32 | Char # => (Int32 | Char)
   # ```
-  def self.|(other : U.class) forall U
+  def |(other : U.class) forall U
```

Comment:
I don't know why this was a class method before, or even why it worked, but with this change it now needs to be an instance method.